targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF52832/TARGET_MTB_UBLOX_NINA_B1/PinNames.h

```diff
@@ -0,0 +1,139 @@
+#ifndef MBED_PINNAMES_H
```

Comment:
can you add license header file please here on the top?

Reply:
@0xc0170 .. Thanks. Done!